templates/package.json

```diff
@@ -6,8 +6,25 @@
     "scripts": {
         "check-sensitive": "npx eslint --plugin 'no-secrets' --cache --ignore-pattern 'package.json' --ignore-pattern 'package-lock.json'",
         "precommit": "npm run check-sensitive && lint-staged",
-        "version": "../.github/scripts/pkg-version.sh core-template"
+        "version": "bash ../.github/scripts/pkg-version.sh template-sync && bash ../.github/scripts/pkg-version.sh core-template"
     },
+    "templates": [
+        "blazor-base/csharp/default",
+        "bot/csharp/default",
+        "bot/js/default",
+        "bot/ts/default",
+        "bot-msgext/js/default",
+        "bot-msgext/ts/default",
+        "function-base/js/default",
+        "function-base/ts/default",
+        "function-trigger/js/HTTPTrigger",
+        "function-trigger/ts/HTTPTrigger",
```

Comment:
function-trigger**s**
Image I add a new template but I add a wrong path to this list. Can we setup test to prevent this kind of error?

Reply:
You own the templates, please add test cases, thanks!